src/test/java/net/datafaker/service/FakeValuesServiceTest.java

```diff
@@ -35,8 +39,11 @@
 
 class FakeValuesServiceTest extends AbstractFakerTest {
 
-    private static final Long MILLIS_IN_AN_HOUR = 1000 * 60 * 60L;
-    private static final Long MILLIS_IN_A_DAY = MILLIS_IN_AN_HOUR * 24;
+    private static final Long SECONDS_IN_AN_HOUR = 60 * 60L;
+    private static final Long SECONDS_IN_A_DAY = SECONDS_IN_AN_HOUR * 24;
```

Comment:
Are there not java constants for this in ChromoUnit or TimeUnit? Or couldn't it be simplified to like `TimeUnit.HOURS.toSeconds(1)` and `TimeUnit.DAYS.toSeconds(1)`

Reply:
good point, thanks
now `TimeUnit` is used for that